admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,82 @@
+===================
+ownCloud User Types
+===================
+
+ownCloud supports six user types.
```

Comment:
It's actually not about technical support but about the design concept behind ownCloud and enabling different roles to be taken by different people, e.g. for small installations "ownCloud Admin", "ownCloud Group Admin", "Storage Admin", "DB Admin", "Auditor" etc. might be one person but for larger installations people want/need to split those roles/responsibilities on different people which we need to take care of in the design of the software.